.travis.yml

```diff
@@ -15,8 +15,8 @@ before_script:
   # remove following files to exclude from awesome_bot link checks because '127.0.0.'... links cause failure even when whitelisted
   - rm docs/guides/quickstart.rst.md docs/guides/deployment.rst.md
   - gem install awesome_bot
-  # run dead link checks excluding white-listed links
-  - awesome_bot --allow-redirect --allow-dupe docs/**/*.md README.md --white-list http://127.0.0.1:8080,http://myhoodieapp.com,https://pouchdb.com/adapters,https://hoodie-app-tracker-randomxyz.now.sh/,https://my-tracker-app.now.sh,https://hapijs.com/tutorials/,https://opencollective.com
+  # run dead link checks but silence 404 failures (due to false postives), redirect, dupes and excluding white-listed links
+  - awesome_bot --allow 404 --allow-redirect --allow-dupe docs/**/*.md README.md --white-list http://127.0.0.1:8080,http://myhoodieapp.com,https://pouchdb.com/adapters,https://hoodie-app-tracker-randomxyz.now.sh/,https://my-tracker-app.now.sh,https://hapijs.com/tutorials/,https://opencollective.com
```

Comment:
I don’t think it’s only 404s. For example jsbin tends to go down from time to time which results in a connection error, I don’t think it’s treated as 404 in that case. Any thought?